CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ We use _breaking :warning:_ to mark changes that are not backward compatible (re
 ### Added
 
 - [#4299](https://github.com/thanos-io/thanos/pull/4299) Tracing: Add tracing to exemplar APIs.
+- [#4327](https://github.com/thanos-io/thanos/pull/4327) Add env substitution to PathOrContent flag.
```

Comment:
The changelog for the users' perspective, so it'd be nice to tell user that they can use env variable for certain flag argument.s